pkg/metrics/delivery_reporter.go

```diff
@@ -155,11 +155,11 @@ func (r *DeliveryReporter) reportEventProcessingTime(ctx context.Context, end ti
 }
 
 func filterTypeValue(v string) string {
-	if v != "" {
-		return v
+	if v == "" {
+		// the default value if the filter attributes are empty.
+		return unspecifiedFilterType
 	}
-	// the default value if the filter attributes are empty.
-	return "any"
+	return EventTypeMetricValue(v)
```

Comment:
It shouldn't be necessary to transform the filter type since the cardinality is already limited by the cardinality of triggers.

Reply:
Across all GCP projects? Doesn't the same cardinality issue occur? Even if a single project has a trigger with one custom event type

Reply:
Removed it